lib/Basic/Platform.cpp

```diff
@@ -246,16 +241,16 @@ getArchForAppleTargetSpecificModuleTriple(const llvm::Triple &triple) {
   auto tripleArchName = triple.getArchName();
 
   return llvm::StringSwitch<StringRef>(tripleArchName)
-              .Cases("arm64", "aarch64", "arm64")
-              .Cases("x86_64", "amd64", "x86_64")
-              .Cases("i386", "i486", "i586", "i686", "i786", "i886", "i986",
-                     "i386")
-              .Cases("unknown", "", "unknown")
-  // These values are also supported, but are handled by the default case below:
-  //          .Case ("armv7s", "armv7s")
-  //          .Case ("armv7k", "armv7k")
-  //          .Case ("armv7", "armv7")
-              .Default(tripleArchName);
+      .Cases("arm64", "aarch64", "arm64")
+      .Cases("x86_64", "amd64", "x86_64")
+      .Cases("i386", "i486", "i586", "i686", "i786", "i886", "i986", "i386")
+      .Cases("unknown", "", "unknown")
+      // These values are also supported, but are handled by the default case
+      // below:
+      //          .Case ("armv7s", "armv7s")
```

Comment:
Likewise here, please restore the original indentation as these are unrelated whitespace changes (and also cause the code in the comments to fall out of alignment with the lines above).

Reply:
I literally did that, and this is what happens every time.